### PR TITLE
Add core weekly 48

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2020-12-02-coreweekly-48-2020.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+The past week has seen many major releases for the PHP ecosystem: [php 8.0](https://www.php.net/releases/8.0/), [xdebug 3](https://xdebug.org/announcements/2020-11-25), [Flysystem 2](https://flysystem.thephpleague.com/v2/docs/what-is-new/) and multiple versions of [Symfony](https://symfony.com/blog/a-week-of-symfony-726-23-29-november-2020) including the latest Symfony3 release: v3.4.47.
+
+We had high hopes to be able to add to this list PrestaShop 1.7.7.0 but unfortunately this did not happen.
+
+Thanks to the huge work of QA Team, multiples issues have been identified in the upgrade process from previous 1.7 versions to 1.7.7.0. As the ability to upgrade is a major keystone for security and stability, we could not release PrestaShop 1.7.7.0 with such issues.
+
+**QA Team has validated the 1.7.7.0 release**. The only missing piece to release it is to validate the stability of the upgrade process, and that is a tough task as there are a great number of paths to be explored.
+
+Iformations about the release status continue to be provided on [the release tracking issue](https://github.com/PrestaShop/PrestaShop/issues/18647) for 1.7.7.0.
```
Comment (ttoine) [inline news/_posts/core-weekly/2020-12-02-coreweekly-48-2020.md]:
```suggestion
Informations about the release status continue to be provided on [the release tracking issue](https://github.com/PrestaShop/PrestaShop/issues/18647) for 1.7.7.0.
```

**Review by ttoine — COMMENTED** [review]
just a typo to fix.

File: news/_posts/core-weekly/2020-12-02-coreweekly-48-2020.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 23th to Sunday 29th of November 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+The past week has seen many major releases for the PHP ecosystem: [php 8.0](https://www.php.net/releases/8.0/), [xdebug 3](https://xdebug.org/announcements/2020-11-25), [Flysystem 2](https://flysystem.thephpleague.com/v2/docs/what-is-new/) and multiple versions of [Symfony](https://symfony.com/blog/a-week-of-symfony-726-23-29-november-2020) including the latest Symfony3 release: v3.4.47.
```
Comment (Progi1984) [inline news/_posts/core-weekly/2020-12-02-coreweekly-48-2020.md:22]:
```suggestion
The past week has seen many major releases for the PHP ecosystem: [PHP 8.0](https://www.php.net/releases/8.0/), [Xdebug 3](https://xdebug.org/announcements/2020-11-25), [Flysystem 2](https://flysystem.thephpleague.com/v2/docs/what-is-new/) and multiple versions of [Symfony](https://symfony.com/blog/a-week-of-symfony-726-23-29-november-2020) including the latest Symfony 3 release: v3.4.47.
```

**Review by Progi1984 — COMMENTED** [review]
Small feedback

**Comment by matks** [comment]
I close the PR as maybe some other announcements will be soon done

**Pull request closed without merging**